src/lib/prepareOas.ts

```diff
@@ -178,26 +183,23 @@ export default async function prepareOas(
   debug('👆👆👆👆👆 finished logging spec 👆👆👆👆👆');
   debug(`spec type: ${specType}`);
 
+  if (opts.title) {
+    debug(`renaming title field to ${opts.title}`);
+    api.info.title = opts.title;
```

Comment:
I was wondering if this would clash with Swagger specs and not work there but thankfully `info.title` exists and is required. https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#info-object

Reply:
oof yeah the `.info.title` type appeared to work properly when writing this so I didn't even consider that 😬